ibm_catalog.json

```diff
@@ -43,71 +43,88 @@
             ],
             "flavors": [
                 {
-                    "label": "Advanced",
-                    "name": "advanced",
+                    "label": "Cluster-with-LSF-v10.1.0.14",
+                    "name" : "Cluster-with-LSF-v10.1.0.14",
                     "install_type": "fullstack",
                     "working_directory": "solutions/hpc",
                     "compliance": {
                         "authority": "scc-v3",
                         "profiles": [
                             {
                                 "profile_name": "IBM Cloud Framework for Financial Services",
-                                "profile_version": "1.5.0"
+                                "profile_version": "1.6.0"
                             }
                         ]
                     },
                     "release_notes_url": "https://cloud.ibm.com/docs/allowlist/hpc-service?topic=hpc-service-release-notes",
                     "configuration": [
                         {
-                            "key": "ibmcloud_api_key",
-                            "type": "password",
-                            "description": "IBM Cloud API Key that will be used for authentication in scripts run in this module. Only required if certain options are required.",
-                            "required": true
+                            "key": "ibmcloud_api_key"
                         },
                         {
-                            "key": "nsd_details",
-                            "type": "array",
-                            "default_value": null,
-                            "description": "Storage scale NSD details",
-                            "hidden": true
+                            "key": "resource_group"
+                        },
+                        {
+                            "key": "reservation_id"
+                        },
+                        {
+                            "key": "cluster_id"
```

Comment:
suggest to use the catalog cluster dropdown option here

Reply:
The reservation id and cluster_id is something that the customer will need to pass and those are not available on the account specific.. So we expect customers to pass the reservation id once they have the contract to use our product ! In that case we still need to update the dropdown option ?

Reply:
ah sorry this is not a kube cluster, its a HPC cluster. There is no dropdown for that so ignore